server/resp/src/main/java/org/infinispan/server/resp/operation/GetexOperation.java

```diff
@@ -0,0 +1,59 @@
+package org.infinispan.server.resp.operation;
+
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+
+import org.infinispan.server.resp.Util;
+
+public class GetexOperation {
+   private static final byte[] PERSIST_BYTES = "PERSIST".getBytes(StandardCharsets.US_ASCII);
+
+   public static long parseExpiration(List<byte[]> arguments) {
```

Comment:
We should consolidate this with https://github.com/infinispan/infinispan/blob/main/server/resp/src/main/java/org/infinispan/server/resp/operation/SetOperation.java#L166

Reply:
You mean extract code in one method?

Reply:
I've tracker this in a separate issue to avoid conflicts with other commands in PR queue.
https://issues.redhat.com/browse/ISPN-14888

Reply:
Can we get this in the same PR. Otherwise I feel like we will forget about this JIRA.

Reply:
on a better thought, which part of the code you want to be shared? common code seems pretty minimal to me

Reply:
I mean, they share 4 options but even the syntax checks are different: i.e. one checks for KEEPTTL the other one for PERSIST